propagation/onboarding.md

```diff
@@ -0,0 +1,76 @@
+# Algorithm Onboarding Service
+
+APEx offers an algorithm onboarding service as part of its propagation services to ensure that project results remain
+accessible to the EO community beyond the project's completion. This service aims to sustain and promote the use of
+algorithms developed during ESA projects, ensuring their longevity and continued utilization.
+
+For engagement with stakeholders of the algorithm, APEx collaborates closely with
+the [Stakeholder Engagement Facility](https://esa-sef.eu/). This initiative will provide outreach and training towards
+users, to maximize the uptake of the onboarded algorithms.
```

Comment:
not sure if we should prominently mention SEF here, its more an esa internal thing I think

Reply:
The aim of this statement was to clarify the relationship with sef. It felt important, because for outsiders it is often difficult to understand how these projects relate to each other, and it makes us more attractive if users know that apex compliant services will receive this kind of support.
Of course, happy to adjust if I simply understood this entirely wrong.

Reply:
we could take the opportunity to define the different types of APEx users (e.g. ESA project, SEF, and public) 
we should orientate the documentation on that direction or at least guide them and they can choose to avoid misunderstandings and disappointments

Reply:
Maybe we can expand the landing page, as discussed in https://github.com/ESA-APEx/apex_documentation/pull/30? I'm not sure if it is relevant to dedicate documentation specifically to the SEF as, from a documentation point of view, they would access our services similar to other ESA projects.